.travis.yml

```diff
@@ -11,27 +11,33 @@ os:
   - linux
   - osx
 
+sudo: required # needed for trusty
+dist: trusty # needed for Raqm
+
 before_install:
   - if [ "$TRAVIS_OS_NAME" = "linux" ];
     then
+      sudo apt-add-repository -y ppa:as-bahanta/raqm;
```

Comment:
Also even if Raqm was added to Debian and Ubuntu today, it would be years before it reaches Travis, so the PPA will be needed for sometime anyway.